remove env-filter, as it's not in Debian right now due packaging problems,
drop this patch when it's back.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -255,7 +255,6 @@
 features = [
     "std",
     "fmt",
-    "env-filter",
 ]
 
 [features]
